ts/features/itwallet/common/components/discoveryBanner/ItwDiscoveryBannerOnboarding.tsx

```diff
@@ -41,24 +48,44 @@ const ItwDiscoveryBannerOnboarding = () => {
     }, [trackBannerProperties, isBannerRenderable])
   );
 
-  if (!isBannerRenderable) {
-    return null;
-  }
-
   const handleOnPress = () => {
     trackItWalletBannerTap(trackBannerProperties);
     navigation.navigate(ITW_ROUTES.MAIN, {
       screen: ITW_ROUTES.DISCOVERY.INFO
     });
   };
 
+  const shouldRender = pipe(
+    O.fromNullable(isWalletRemotelyActive),
+    O.fold(
+      () => false,
+      () => isBannerRenderable
+    )
+  );
+
+  const bannerConfig = {
+    onboarding: {
+      content: I18n.t("features.itWallet.discovery.banner.home.content")
+    },
+    reactivating: {
+      content: I18n.t(
+        "features.itWallet.discovery.banner.onboardingActive.content"
+      )
+    }
+  };
+
+  const bannerType = isWalletRemotelyActive ? "reactivating" : "onboarding";
+  const { content } = bannerConfig[bannerType];
+
+  if (!shouldRender) {
+    return null;
+  }
```

Comment:
I think it's safe to use `isWalletRemotelyActive` directly. What do you think?

```suggestion
  if (!isWalletRemotelyActive) {
    return null;
  }
```

Reply:
Going this way we are skipping the check on the selector that checks this condition:  `!itwLifecycleIsValidSelector(state) && isItwEnabledSelector(state)`. Isn't that a problem?

Reply:
You are right! I misread the `pipe` function above!

Reply:
You are right! But there is something that I do not understand in the conditional render logic.
If `isWalletRemotelyActive` is `some`, we take the value of `isBannerRenderable` and if it's `none` we return `false`, skipping the `isBannerRenderable` value. 
I think this is not correct. 
We need to show the banner even if `isWalletRemotelyActive` is not defined for some reason. Am I missing something?

Reply:
Yes you are right, addressed here https://github.com/pagopa/io-app/pull/6833/commits/7555f6e6fed8d3097d0e7b5eb773e2399f4517d8. Thanks!